src/panels/lovelace/hui-glance-elegant-card.js

```diff
@@ -0,0 +1,163 @@
+        }
+        img {
+          width: 100%;
+          height: auto;
+          border-radius: 2px;
+        }
+        .box {
+          @apply --paper-font-common-nowrap;
+          position: absolute;
+          left: 0;
+          right: 0;
+          bottom: 0;
+          background-color: rgba(0, 0, 0, 0.3);
+          padding: 4px 16px;
+          font-size: 16px;
+          line-height: 40px;
+          color: white;
+          border-bottom-left-radius: 2px;
+          border-bottom-right-radius: 2px;
+          display: flex;
+          justify-content: space-between;
+        }
+        .box .title {
+          font-weight: 500;
+        }
+        paper-icon-button, iron-icon {
+          color: #A9A9A9;
+        }
+        paper-icon-button.state-on, iron-icon.state-on {
+          color: white;
+        }
+        iron-icon {
+          padding: 8px;
+        }
+        .error {
+          background-color: red;
+          color: white;
+          text-align: center;
+        }
+      </style>
+
+      <ha-card>
+        <img src="[[config.image]]">
+        <div class="box">
+          <div class="title">[[config.title]]</div>
+          <div>
+            <template is="dom-repeat" items="[[_entities]]">
+              <template is="dom-if" if="[[_showEntity(item, hass.states)]]">
+                <paper-icon-button
+                  on-click="_callService"
+                  class$="[[_computeClass(item, hass.states)]]"
+                  icon="[[_computeIcon(item, hass.states)]]"
+                  title="[[_computeTooltip(item, hass.states)]]"
+                ></paper-icon-button>
+              </template>
+            </template>
+          </div>
+        </div>
+        <template is="dom-if" if="[[_error]]">
+          <div class="error">[[_error]]</div>
+        </template>
+      </ha-card>
+    `;
+  }
+
+  static get properties() {
+    return {
+      hass: Object,
+      config: Object,
+      _entities: {
+        type: Array,
+        computed: '_computeEntities(config)'
+      },
+      _error: String
+    };
+  }
+
+  getCardSize() {
+    return 3;
+  }
+
+  _computeEntities(config) {
+    if (config && config.entities && Array.isArray(config.entities) && config.image) {
+      this._error = null;
+      return config.entities;
+    }
+    this._error = 'Error in card configuration.';
+    return [];
+  }
+
+  _showEntity(entityId, states) {
+    return entityId in states;
+  }
+
+  _computeIcon(entityId, states) {
+    return stateIcon(states[entityId]);
+  }
+
+  _computeClass(entityId, states) {
+    return STATES_ON.includes(states[entityId].state) ? 'state-on' : '';
+  }
+
+  _computeTooltip(entityId, states) {
+    return `${computeStateName(states[entityId])}: ${computeStateDisplay(this.localize, states[entityId])}`;
+  }
+
+  _callService(ev) {
+    const entityId = ev.model.item;
+    const domain = computeDomain(entityId);
+
+    if (DOMAIN_SENSORS.includes(domain) || this.config.force_dialog) {
```

Comment:
I don't like this. For good UX, the action should be consistent tapping on different icons. I guess we can't put spacing between sensors and the other icons because the user defines order?